src/components/workflowEditor/nodes/StaticNode.tsx

```diff
@@ -17,12 +17,19 @@ export interface StaticNodeProps {
     sourceType: string
     regex?: string
     primaryBranchAfterRegex?: string
+    to?: string
+    handleGoToWorkFlowEditor?: (...args) => void
 }
 
 export class StaticNode extends Component<StaticNodeProps> {
     renderCardContent() {
         return (
-            <div className="workflow-node workflow-node--static">
+            <div
+                className={`workflow-node workflow-node--static ${
+                    this.props.branch === 'Not Configured' ? 'cursor workflow-node--dash' : ''
+                }`}
+                onClick={this.props.branch === 'Not Configured' ? this.props.handleGoToWorkFlowEditor : null}
```

Comment:
Can we move this check in the function `this.props.handleGoToWorkFlowEditor`

Reply:
done